### DISCUSSION TITLE
How to wipe all installed OSs within PINN?

### DISCUSSION THREAD

**Discussion by bpdronkers** [procount/pinn#707] [Q&A]
Is there an easy way within PINN to wipe all installed OSs (including resetting all partitions) and selecting new OSs for PINN to download and install? Or is the easiest way to just reformat the SD card with my Ubuntu laptop and copy over a fresh install of PINN? It would be nice if this is something I could do directly within PINN.

**Comment by procount** [comment]
The install function is destructive so it will wipe everything already installed.
Actually there are a few dialog windows to go though to stop you doing this accidentally!
Use the clear button to clear the selection of already installed Oses before choosing your new set to install.

If you want to replace an OS without removing the others, you need to first select your new Os(es) then go to the maintenance menu and select the Os(es) you want replaced, then hit the replace button and match up the new Oses with the old ones.

**Reply by bpdronkers** [reply]
That makes sense! And to your point, if I want to replace an existing install, then I use the replace button/process. Thanks for your quick help!
